.github/scripts/constants.sh

```diff
@@ -1,7 +1,8 @@
 # Igluctl
 INSTALL_DIR="/tmp"
-VALIDATOR_V="0.7.2"
-VALIDATOR_URI="https://github.com/snowplow-incubator/igluctl/releases/download/${VALIDATOR_V}/igluctl"
+VALIDATOR_V="0.10.1"
+VALIDATOR_ZIP="igluctl_${VALIDATOR_V}.zip"
+VALIDATOR_URI="https://github.com/snowplow-incubator/igluctl/releases/download/${VALIDATOR_V}/$VALIDATOR_ZIP"
```

Comment:
Not sure how long GitHub maintains redirecting old repo addresses, it works now but the repo lives under `snowplow` now, you might want to change that to
```
VALIDATOR_URI="https://github.com/snowplow/igluctl/releases/download/${VALIDATOR_V}/$VALIDATOR_ZIP"
```